docs/faq.md

```diff
@@ -80,3 +80,23 @@ ____
 ### Q: How can I get my CAT listed in Chia's wallet GUI?
 
 **A:** Sometime in the first half of 2022, we plan to release a process to apply for your CAT to be listed in our wallet GUI. Keep in mind that this will be a tightly-controlled process, where few CATs will be accepted. But don't let that stop you from creating your own CATs -- they will still work, even if they are not listed in our GUI.
+
+-----
+
+### Q: What is hinting?
+
+**A:** Before we discuss _what_ hinting is, it's important to understand _when_ it applies. As explained in the [Conditions](/docs/coins_spends_and_wallets#conditions "Condition codes") section, the syntax for the `CREATE_COIN` condition is `(51 puzzlehash amount (memo memo ...))`. The final argument (in parentheses, after `amount`) is the _memo_ list. This list can be arbitrarily long, and its interpretation depends on how it is structured. There are three possibilities:
+
+* If the memo list doesn't exist, then the CREATE_COIN condition proceeds without it. In other words, the list is optional
+* If the first `memo` argument is not exactly 32 bytes, then the CREATE_COIN condition proceeds without it. In this case, the entire memo list is treated as a no-op
+* If the first `memo` argument is exactly 32 bytes, then it is treated as a hint and subsequent arguments are ignored
+
+#### What is a hint?
+A hint is an extra value that is mapped to a coin, which is then indexed by the full nodes. If a coin includes a hint, then a wallet that sees the coin will have one extra piece of information to determine how the coin should be interpreted.
```

Comment:
I don't think this is incorrect, but I think it would be good to highlight that it gives wallets the ability to look up a coin by the hint rather than just use it for additional information once they already have the coin.

Reply:
@Quexington I added your suggestion and reorganized it to make it more readable.
@roybotbot If Quex thinks it's good to go, then yes merge please.